<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyQtTabWidget2</class>
 <widget class="QWidget" name="MyQtTabWidget2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyQtTabWidget2</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget"/>
   </item>
   <item>
    <widget class="QPushButton" name="btnAdd">
     <property name="text">
      <string>添加TabWidget Page</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnDelete">
     <property name="text">
      <string>删除TabWidget Page</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
